src/imitation/scripts/train_preference_comparisons.py

```diff
@@ -52,7 +52,7 @@ def train_preference_comparisons(
     agent_kwargs: Mapping[str, Any],
     gatherer_kwargs: Mapping[str, Any],
     allow_variable_horizon: bool,
-) -> Mapping[str, Any]:
+) -> Mapping[str, Mapping[str, float]]:
```

Comment:
I don't think that's correct, the `reward_loss` and `reward_accuracy` keys currently live directly in the top-level dict (we could change that though)

Reply:
Good catch, I'll revert this.